build.gradle

```diff
@@ -47,6 +49,12 @@ allprojects {
         sourceCompatibility = 1.8
         tasks.withType(JavaCompile) {
             options.compilerArgs += ['-Werror', '-Xlint:deprecation']
+            options.errorprone {
+                check("Slf4jLogsafeArgs", CheckSeverity.OFF)
```

Comment:
`disable` instead of off

Reply:
sadly this needs 0.8 which would require gradle 5.2